i18n/SpanishMessages.scala

```diff
@@ -0,0 +1,237 @@
+  override val addAtomUsageText = """Argumentos no válidos. Uso: <pre>/addrss $account_name</pre> Por ejemplo, debes usar el siguiente comando:
+    |<pre>/addrss https://feedforall.com/sample.xml</pre>""".stripMargin
+  override val enterReddit = "EL Bot puede monitorear publicaciones en un SubReddit también como publicaciones de usuarios de Reddit. Ingresa el nombre del Subreddit, el usuario de Reddit o la URL."
+  override val addRedditUsageText = """Argumentos no válidos. Uso: <pre>/addreddit $subreddit</pre> Por ejemplo, debes usar los siguientes comandos:
+    |<pre>/addreddit cristianoronaldo</pre> o
+    |<pre>/addreddit https://www.reddit.com/r/cristianoronaldo/</pre> o
+    |<pre>/addreddit https://www.reddit.com/user/jdegoes</pre>""".stripMargin
+  override val enterYouTubeText = "Ingresa el texto para buscar canales de YouTube."
+  override val addYouTubeUsageText = """Argumentos no válidos. Uso: <pre>/addyoutube search_query</pre> Por ejemplo, debes usar el siguiente comando:
+    |<pre>/addyoutube Cristiano Ronaldo</pre>""".stripMargin
+  override val noYouTubeChannels = "No se encontraron canales de YouTube con lo que buscas"
+  override val chooseYouTubeChannels = "Se encontraron los siguientes canales. Elige el canal al que quieras suscribirte:"
+  override val enterTwitchText = "Ingresa el canal de Twitch o la URL"
+  override val enterTwitchUsageText = """Argumentos no válidos. Uso: <pre>/addtwitch $channel_name</pre> Por ejemplo, debes usar los siguientes comandos:
+    |<pre>/addtwitch GSL</pre> o
+    |<pre>/addtwitch https://www.twitch.tv/gsl</pre>""".stripMargin
+
+  override val noLink = "No se encontró ningún texto"
+  override val backToBotSettingsButton = "<< Volver a la configuración"
+
+  override def botSettings(channelName: Option[String]) =
+    header(channelName) + "Configuración del Bot:"
+
+  override val onlyForAdmins = "Solo los administradores puede ejecutar este comando"
+  override val permissionsButton = "Permisos"
+  override val restrictButton = "Restringir"
+  override val disableRestrictionButton = "Deshabilitar restricciones"
+  override val unrestrictedMode = "Todos los miembros de este grupo pueden agregar/borrar subscripciones. Haz click en el botón de abajo para " +
+    "restringir estos comandos y requerir permisos de administrador."
+  override val restrictedMode = "Solo los administradores pueden agregar/borrar suscripciones. Haz click en el botón de abajo para " +
+    "brindar acceso a todos los miembros de este grupo."
+  override val noPermissions = "Solo los administradores pueden ejecutar esta acción!"
+
+  override val wordFilter = "Filtro de Palabras"
+  override val editWordfilterButton = "Editar Filtros"
+  override val removeWordfilterButton = "Quitar Filtros"
+  override def noStopWords(channelName: Option[String]) =
+    header(channelName) +
+      "El Filtro de Palabras no está configurado. Use el Filtro de Palabras
```

Comment:
Probably you should add `" + ` at the end of file and open quote on next line.

Reply:
Oh, yes, I don't have a proper linter for Scala, I missed it. Fixed now.